### ISSUE TITLE
Request methods

### ISSUE THREAD

**Issue by itm-ape** [johannschopplich/nuxt-api-party#2]
Hello I'm looking at this module and it seems exactly like something I could use for my project. Looking at the code though it doesn't seem like it's possible to perform any other request methods than 'GET' requests. Is this something that will be added in the near feature? Otherwise this module looks very nice. Amazing work!

**Comment by johannschopplich** [comment]
I can put it on the roadmap. Would you mind providing an example of your use-case?

**Comment by itm-ape** [comment]
I'm working with a backend where I need to perform POST requests to an endpoint while parsing a JSON body. Maybe I'm just misunderstanding how the module works, but it doesn't seem possible to specify the HTTP request method and pass a body.

As an example this Postman request I would like to be able to perform from my frontend using your module 
![image](https://user-images.githubusercontent.com/78532174/205029007-527b261d-9dbd-4643-8652-0060c7840e38.png)

Hope it makes sense

**Comment by johannschopplich** [comment]
Now available in v0.4.0+!

Example:

```vue
<script setup lang="ts">
const data = await $party(
  'posts',
  {
    method: 'POST',
    body: {
      foo: 'bar'
    },
  }
)
</script>

<template>
  <div>
    <h1>{{ data?.title }}</h1>
  </div>
</template>
```
